lib/configs/react.js

```diff
@@ -8,8 +8,14 @@ module.exports = {
   plugins: ['github', 'jsx-a11y'],
   extends: ['plugin:jsx-a11y/recommended'],
   rules: {
-    'github/a11y-no-generic-link-text': 'error',
+    'jsx-a11y/role-supports-aria-props': 'off', // Override with github/role-supports-aria-props until https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/910 is resolved
     'github/role-supports-aria-props': 'error',
-    'jsx-a11y/role-supports-aria-props': 'off',
+    'jsx-a11y/no-aria-hidden-on-focusable': 'error',
+    'jsx-a11y/anchor-ambiguous-text': [
+      'error',
+      {
+        words: ['this', 'more', 'read here', 'read more'],
```

Comment:
Rules in addition to the defaults.

[anchor-ambiguous-text rule options](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/anchor-ambiguous-text.md#rule-options)